features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreViewModel.kt

```diff
@@ -206,6 +208,7 @@ class ExploreViewModel @Inject constructor(
     val screenState: LiveData<ScreenState>
         get() = _screenState
 
+    var purchaseGiftCardData: Pair<Pair<Coin, Fiat>, Merchant>? = null
```

Comment:
This seems a bit convoluted. Let's create a dedicated data class to store this.

Reply:
Refactored code in Putsches viewmodel

Reply:
Good Point, we have some things like this in DashJ too.